JMS/src/test/java/com/ibm/mq/samples/jms/SampleEnvSetterTest.java

```diff
@@ -0,0 +1,201 @@
+/*
+* (c) Copyright IBM Corporation 2024
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+package com.ibm.mq.samples.jms;
+
+import static org.junit.jupiter.api.Assertions.*;
+import java.lang.reflect.Field;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.BeforeAll;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import java.util.ArrayList;
+import java.util.List;
+
+public class SampleEnvSetterTest {
+
+    static SampleEnvSetter envSetter;
+    
+    @BeforeAll
+    public static void setUp() throws NoSuchFieldException, IllegalAccessException{
```

Comment:
I don't think we should be mocking the endpoints. The point of the test is to verify:
1. That the env setter is able to read the file, whose name and location are specified by an environment variable. 
2. That it allows overrides via environment variables.
3. How it behaves when there are missing values in the env file.
4. How it behaves when the file can't be found.